### DISCUSSION TITLE
How to find out programmatically the list of icons in the launcher screen of Jupyter Lab

### DISCUSSION THREAD

**Discussion by joseberlines** [jupyterlab/jupyterlab#14696] [Q&A]
We have a large JupyterHub in our organization that serves different "flavours" of Jupyter Lab to users depending on the profile of every user.

How can I access in the terminal the list of icons present in the Launcher:

Imagining the launcher showed next I would like to programaticaly being able to have something like:
{"Notebook":["Python 3","LangChain",...],
"COnsole": [...],
"Other": [...]
}

you get the point

![image](https://github.com/jupyterlab/jupyterlab/assets/33203439/e12fa233-d8e1-4352-b5a8-0af382155471)

**Comment by DzmitrySudnik** [comment]
@joseberlines did you figure out how to do it?
